### ISSUE TITLE
DAC seems not to be recognised

### ISSUE THREAD

**Issue by edegraaff** [maxnet/berryboot#526]
Hi, 
Seems with the latest berryboot my hifiverry is not longer supported. Strange enough all drivers seem to be ok, but within volumio the DAC is not recognised.  The Volumio distribution as IMG file is working fine. Seems when i run berryboot with the latest small installer of Debian the DAC is also not recognised. 

dtoverlay=hifiberry-digi 
did work in the past but currently it is not. 

See also this discussion
https://support.hifiberry.com/hc/en-us/community/posts/360008646745-strange-behavior-volumio-and-hifiberry-digi-berryboot

and this one

https://volumio.org/forum/multiboot-volumio2-with-kodi-under-berryboot-t6818-20.html#p50050

regards 

eelco

**Comment by maxnet** [comment]
How did you edit config.txt ?
Using the Berryboot menu editor?

**Comment by edegraaff** [comment]
ettitor on the sd card mounted windows notepad ++

**Comment by edegraaff** [comment]
btw i realize the volumio instance is created by a script. on the other hand if berryboot loads the dac drivers in the right way every mounted linux image should be enabled with the right drivers. am i correct?

btw i editted the config.txt wrom the menu now the hifiberry setting was there visible

**Comment by maxnet** [comment]
>the other hand if berryboot loads the dac drivers in the right way every mounted linux image should be 
>enabled with the right drivers. am i correct?

Yes, provided your config.txt is indeed correct, udev should load modules automatically.

`sudo vcdbg log msg` should confirm your config.txt is correct and it loaded the overlay file correctly.
`lsmod` should show that it loaded the module (although it may be called something different like wm8804 after the name of the chip used, rather than hifiberry).
If it failed loading for whatever reason, `dmesg` may also show messages about it.

**Comment by macmpi** [comment]
@edegraaff thanks to provide logs as per maxnet recommandations.
Also you may want to try adding  `dtparam=i2c_arm=on` in ` config.txt` in case it is not there.
